### DISCUSSION TITLE
Release Candidate 1.6.530x.0

### DISCUSSION THREAD

**Discussion by BlueFinBima** [HeliosVirtualCockpit/Helios#571] [General]
I've pulled together the release notes for 1.6.53xx https://github.com/HeliosVirtualCockpit/Helios/releases/tag/untagged-f16e49466ea8cf3f7095 as best I can.  I think that the StreamDeck stuff could do with some embellishment from a knowledgeable party (@derammo / @Todd1215 ).  
Obviously it would be good for someone to "kick the tyres" (sp) of the candidate.
I'm currently following the wiki entry on preparing a release, but @Todd1215 - you're the expert so I'll defer to you if you think I need to be doing it a different way or something needs to be improved.

**Comment by derammo** [comment]
Seems fine by me.  I would suggest you put your Apache stuff at the top because that seems to be the way most important thing?

We could be more explicit about what "needs a developer" means.    Maybe this?

> This project is in alpha state.  It runs and can support simple scenarios of using a Stream Deck in a Helios profile.  A C# developer will be needed to advance this project any further, since nobody is currently working on it.  Support is available from @derammo for anyone who wants to take this on.

**Reply by BlueFinBima** [reply]
updates made in line with comments.

**Comment by Todd1215** [comment]
I added a few bullet points about the HeliosMacroboard plugin. Other than that it all looks ok to me. I followed the wiki for release as well so we are in line where that is concerned.

**Reply by BlueFinBima** [reply]
Thanks

**Reply by BlueFinBima** [reply]
Verification activities documented here https://github.com/HeliosVirtualCockpit/Helios/issues/576

I plan to make [this](https://github.com/HeliosVirtualCockpit/Helios/releases/tag/untagged-f54314dad325bc76ffc8) public on Saturday unless anyone has any objections.

**Comment by derammo** [comment]
abstaining since I didnt run it.  the report looks lovely
